etc/profile-m-z/parsecd.profile

```diff
@@ -0,0 +1,22 @@
+include parsecd.local
+include globals.local
+
+whitelist ${HOME}/.parsec
+whitelist /usr/share/parsec
+include whitelist-common.inc
+include whitelist-usr-share-common.inc
+
+apparmor
```

Comment:
It's fine and very well you've tested which options work and which don't, like you mention in the accompanying explanations. To keep such valuable info we indeed use comments. Please add those as much as you can.

Reply:
I do not see the need to add a comment here, as I added this in the same way I added private-tmp and both work perfectly and there is nothing to note about this.